addons/sale/wizard/sale_payment_link.py

```diff
@@ -24,6 +24,17 @@ def default_get(self, fields):
             })
         return res
 
+    @api.depends('company_id', 'partner_id', 'currency_id')
+    def _compute_available_acquirer_ids(self):
+        sale_links = self.filtered(lambda link: link.res_model == 'sale.order')
+        super(PaymentLinkWizard, self-sale_links)._compute_available_acquirer_ids(self)
```

Comment:
Mea culpa, the super call shouldn't receive an argument :man_facepalming: 
```suggestion
        super(PaymentLinkWizard, self-sale_links)._compute_available_acquirer_ids()
```